src/dialects/mssql/index.js

```diff
@@ -85,6 +85,15 @@ assign(Client_MSSQL.prototype, {
     })
   },
 
+  prepBindings(bindings) {
+    return map(bindings, (value) => {
+    if ( typeof value === 'number' && value >= 2147483647 || value <= -2147483648){
```

Comment:
the limit for tedious are actually those numbers, kinda hardcoded there : https://github.com/pekim/tedious/blob/50e76efdcd87f687106c722d54c6ea4096f0a127/src/data-type.js#L573